samcli/commands/local/invoke/cli.py

```diff
@@ -31,11 +34,10 @@
     "--event",
     "-e",
     type=click.Path(),
-    default=STDIN_FILE_NAME,  # Defaults to stdin
     help="JSON file containing event data passed to the Lambda function during invoke. If this option "
-    "is not specified, we will default to reading JSON from stdin",
+    "is not specified, no event is assumed. Pass in the value '-' to input JSON via stdin",
```

Comment:
The default value is '-', so there is no need for them to pass '-' to the option `--event` is sufficient.

Reply:
That's not the default behavior anymore. If you do not pass in `--event`, we will not trigger STDIN. That's the point of this change. Unless I'm missing something or made a mistake.

Reply:
My mistake. I thought the default value was still there (which makes no sense and it was removed in the diff). Please ignore my comment.